include/bbp/sonata/population.h

```diff
@@ -226,6 +228,8 @@ class SONATA_API PopulationStorage
 
     PopulationStorage(const PopulationStorage&) = delete;
 
+    PopulationStorage(PopulationStorage&&);
```

Comment:
iirc, @tristan0x showed that move ctors need to be marked `noexcept` for some optimizations to kick in.  Maybe he can comment?

Reply:
That's a good point (I believe he mentioned the knows case of std::vector which will not move in case it's no except, also for the destructor I believe...)
In this case we don't have copy-ctor so that eventual hidden penalty is not there.
Either way I'd gladly mark it as noexcept. But can we ensure so?

Reply:
Cool! Optimizations in!